tools/codegen/gen-config-mapper/api_to_provider_spec_mapper.go

```diff
@@ -0,0 +1,3 @@
+package genconfigmapper
```

Comment:
I added `mapper` because otherwise gen-config and gen-code-spec might not be super clear at a first glance, but not a strong opinion. Will update.

Reply:
slightly related to this, would try to keep directories with same name as packages. Since we are under codegen directory thinking we can have more simple package names/directories: `openapi`, `config`, `codespec`, and `schema` (package I would be adding)

Reply:
updated directory names